squid checks for dbopen, but it seems to exist only in BDB4
--- squid-3.5.4/helpers/external_acl/session/Makefile.am.~1~	2015-05-05 13:13:06.575579651 +0300
+++ squid-3.5.4/helpers/external_acl/session/Makefile.am	2015-05-05 13:13:13.808917002 +0300
@@ -14,4 +14,4 @@
 
 LDADD = \
 	$(COMPAT_LIB) \
-	$(LIB_DB)
+	-ldb
